com.maplerain.telsis_translator_flutter.yml

```
@@ -0,0 +1,38 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: com.maplerain.telsis_translator_flutter
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: telsis_translator_flutter
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+modules:
+  # telsis_translator_flutter
+  - name: telsis_translator_flutter
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - chmod +x telsis_translator_flutter
+      - mv com.maplerain.telsis_translator_flutter.png /app/share/icons/hicolor/256x256/apps/
+      - mv com.maplerain.telsis_translator_flutter.desktop /app/share/applications/
+      - mv com.maplerain.telsis_translator_flutter.metainfo.xml /app/share/metainfo/
+      - mv * /app/
+      - mkdir -p /app/bin
+      - ln -s /app/telsis_translator_flutter /app/bin/telsis_translator_flutter
+    sources:
+      - type: archive
+        url: https://github.com/vivian-ng/telsis_translator_flutter/releases/download/0.2.1/telsis-translator-flutter_0.2.1.tar.gz
```

```suggestion
        url: https://github.com/vivian-ng/telsis_translator_flutter/releases/download/0.2.1/telsis-translator-flutter_0.2.1.tar.gz
        strip-components: 0
```

This is needed given the layout of the archive.